### PR TITLE
Updates design

### PR DESCRIPTION
**Pull request by bvishal8510**

### CODE REVIEW

File: core/urls.py [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
+    # url(r'^practice-session/$', PracticeSessionView.as_view(), name='practice-session'),
+    # url(r'^posters/$', PosterView.as_view(), name='poster'),
+    # url(r'^circulars/$', CircularView.as_view(), name='circular'),
+    # url(r'^video-lecture/$', VideoLectureView.as_view(), name='video-lecture'),
+    # url(r'^contact/$', ContactView.as_view(), name='contact'),
+    # url(r'^workshop/$', WorkshopRegistrationView.as_view(), name='workshop'),
+    # url(r'^event/$', EventRegistrationView.as_view(), name='event'),
+    # url(r'^past-workshop/$', PastWorkshopDetailView.as_view(), name='past-workshop'),
+    # url(r'^uhvpe-program/$', UHVPEProgramView.as_view(), name='uhvpe-program'),
+    # url(r'^developer/$', DeveloperView.as_view(), name='developer'),
```
Comment (apoorvaeternity) [inline core/urls.py]: Remove these comments.

File: core/urls.py [hunk 2]
```diff
@@ -1,33 +1,55 @@
...
-                    ContactView, WorkshopRegistrationView, EventRegistrationView, PastWorkshopDetailView,
-                    UHVPEProgramView, DeveloperView)
+from .views import (IndexView, DirectorMessageView, VisionAndMissionView, EstablishmentView, StructureView, SyllabusView,
+                    SyllabusView, BookAndAuthorsView, UnitWiseNotesView, VideoLecturesView, UnitWisePPTView, UTQuestionPaperView, 
+                    PreviousPaperView, FDPView, StudentsWorkshopView, UpcomingWorkshopView, EventsView, ClubsView,
+                    ImapactStudentView, ImapactFacultyView)
+                    # GuidelineView, SociallyRelevantProjectView,BookView, FAQView, JourneySoFarView, ImpactView, 
+                    # FuturePlansView, NodalCenterView, PresentationView, NoteView, QuestionPaperView, PracticeSessionView, 
+                    # PosterView, CircularView, VideoLectureView, ContactView, WorkshopRegistrationView, EventRegistrationView, 
+                    # PastWorkshopDetailView, UHVPEProgramView, DeveloperView)
```
Comment (apoorvaeternity) [inline core/urls.py]: Remove these comments.

File: core/static/js/numscroller-1.0.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    function numberRoller(slno){
+            var min=$('.roller-title-number-'+slno).attr('data-min');
+            var max=$('.roller-title-number-'+slno).attr('data-max');
+            var timediff=$('.roller-title-number-'+slno).attr('data-delay');
+            var increment=$('.roller-title-number-'+slno).attr('data-increment');
+            var numdiff=max-min;
+            var timeout=(timediff*1000)/numdiff;
+            //if(numinc<10){
+                //increment=Math.floor((timediff*1000)/10);
+            //}//alert(increment);
```
Comment (apoorvaeternity) [inline core/static/js/numscroller-1.0.js]: Remove these comments.

File: core/static/js/numscroller-1.0.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            numberRoll(slno,min,max,increment,timeout);
+            
+    }
+    function numberRoll(slno,min,max,increment,timeout){//alert(slno+"="+min+"="+max+"="+increment+"="+timeout);
```
Comment (apoorvaeternity) [inline core/static/js/numscroller-1.0.js]: Same as above.

**Comment by bvishal8510** [comment]
@apoorvaeternity please review and merge the changes

**Pull request merged**